neuron/src-bash/neuron-search

```diff
@@ -7,8 +7,14 @@ SEARCHFROMFIELD=${3}
 EXTENSIONS=${4}
 OPENCMD=$(envsubst -no-unset -no-empty <<<${5})
 cd ${NOTESDIR}
-rg --no-heading --no-line-number --with-filename --sort path "${FILTERBY}" -g "${EXTENSIONS}" \
+
+{    rg --no-heading --no-line-number --with-filename --sort path "${FILTERBY}" -g "${EXTENSIONS}" \
+  && rg --no-heading --no-line-number --with-filename --sort path "${FILTERBY}" -g "${EXTENSIONS}" \
+     --files-without-match | awk '{ printf "%s:!title: %s\n", $0, $0}'; } \
+  | ( [ ${FILTERBY} ] && LC_ALL=C sort -t: -k1,2 -r || cat ) \
```

Comment:
Why is `LC_ALL` being set here? Is that necessary for this PR?

Reply:
Also, what does the `sort` do here? A search for `link` shows: 

![image](https://user-images.githubusercontent.com/3998/97367403-663b2f00-187f-11eb-900a-6dca7bfebc2b.png)

which appears to be the inverse of "select the most matching item" (which is 'Uplink Tree' IIUC).

Reply:
Okay, that sort order is the same as `master`, so I guess that's irrelevant. Also, the original command had a `--sort path`, which I think sorts by filename.

Reply:
I ran into a problem with `sort` when it complained about "illegal byte sequence". Most likely one of the files was indeed "bad", but the failure mode was ugly in that a normal user would have no idea what's going on, and the `LC_ALL` fixed that. If there is a way to point out what file has the "illegal byte sequence" the user could just remove or fix that file, but error reporting in this context looks hard, so I thought it better to just avoid the problem. 

But I have no strong opinion on leaving it in or removing it.